### PR TITLE
Add testing

### PR DESCRIPTION
**Pull request by brymut**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+flake8:
+	@flake8 --exclude=.git *.py tcms_pytest_plugin tests
+
+.PHONY: pylint
+pylint:
+	pylint -d missing-docstring tcms_pytest_plugin/ tests/
+
+.PHONY: test
+test:
+	pytest --kiwitcms
```
Comment (atodorov) [inline Makefile:15]: Missing newline at EOF.

File: README.rst [hunk 2]
```diff
@@ -69,13 +69,15 @@ the coverage at least stays the same before you submit a pull request.
 License
 -------
 
-Distributed under the terms of the `GNU GPL v3.0`_ license, "kiwitcms-pytest-plugin" is free and open source software
+Distributed under the terms of the `GNU GPL v3.0`_ license,
+"kiwitcms-pytest-plugin" is free and open source software
```
Comment (atodorov) [inline README.rst:73]: Does README need to be updated ?

File: requirements.txt [hunk 3]
```diff
@@ -1,2 +1,2 @@
 pytest
-tcms-api==8.6.0
+tcms-api==9.0
```
Comment (atodorov) [inline requirements.txt]: This is already in master.

**Review by atodorov — CHANGES_REQUESTED** [review]
Few changes requested.

File: README.rst (hunk 2 above)
Comment (brymut) [reply README.rst:73]: I'm getting `README.rst:72: D001 Line too long` from Doc8

**Pull request merged**